src/components/StyledInput.tsx

```diff
@@ -52,9 +52,9 @@ const StyledInput: FC<Props> = ({
           type={type}
           className={css(
             styles.input,
-            style,
             isFocused && styles.focused,
-            !!error && styles.error
+            !!error && styles.error,
+            style
```

Comment:
I'm sorry I mistaken u yesterday and it should be like previous in this order: styles.input, styles, focusStyles, errorStyles